softdevice_controller/include/sdc.h

```diff
@@ -214,85 +214,85 @@ typedef union
  *
  * To change the default configuration, update @ref
- * BLE_CONTROLLER_DEFAULT_RESOURCE_CFG_TAG. To create or update a new
+ * SDC_DEFAULT_RESOURCE_CFG_TAG. To create or update a new
  * configuration, provide another resource_cfg_tag.
  *
  * @note The application can set config_type to @ref
- *       BLE_CONTROLLER_CFG_TYPE_NONE to obtain the required memory size for the
+ *       SDC_CFG_TYPE_NONE to obtain the required memory size for the
  *       current configuration in bytes.
  *
  * @note Resource configuration can only be performed prior to calling @ref
- *       ble_controller_enable(). However, the current configuration may be
- *       changed after enabling the BLE Controller.
+ *       sdc_enable(). However, the current configuration may be
+ *       changed after enabling the SoftDevice Controller.
  *
  * @param[in]  config_tag       Configuration tag.
- * @param[in]  config_type      Configuration type. @sa BLE_CONTROLLER_CFG_TYPE.
+ * @param[in]  config_type      Configuration type. @sa SDC_CFG_TYPE.
  * @param[in]  p_resource_cfg   Configuration to be changed.
  *
  * @returns Required memory size for the current configuration in bytes.
  * @retval -NRF_EOPNOTSUPP    Unsupported configuration
  * @retval -NRF_EINVAL        Invalid argument provided
  */
-int32_t ble_controller_cfg_set(uint8_t config_tag,
-                               uint8_t config_type,
-                               ble_controller_cfg_t const * p_resource_cfg);
+int32_t sdc_cfg_set(uint8_t config_tag,
+                    uint8_t config_type,
+                    sdc_cfg_t const * p_resource_cfg);
 
 
-/** @brief Enable the BLE Controller
+/** @brief Enable the SoftDevice Controller
  *
  * After this function is called, the application may utilize HCI APIs.
  *
  * @param[in] callback  The callback will be executed when HCI data or and HCI
  *                      event is available. The callback will be executed in
  *                      the same context as mpsl_low_priority_process.
- *                      @sa @ref hci_evt_get() and @ref hci_data_get().
+ *                      @sa @ref sdc_hci_evt_get() and @ref sdc_hci_data_get().
  * @param[in]  p_mem    Provide memory for the current resource configuration. If
  *                      custom resource configurations are used, use the value
- *                      returned from @ref ble_controller_cfg_set().
+ *                      returned from @ref sdc_cfg_set().
  *
  * @retval 0            Success
  * @retval -NRF_EINVAL  Invalid argument provided
  */
-int32_t ble_controller_enable(ble_controller_callback_t callback,
-                              uint8_t * p_mem);
+int32_t sdc_enable(sdc_callback_t callback,
+                   uint8_t * p_mem);
 
 
-/** @brief Disable the BLE Controller
+/** @brief Disable the SoftDevice Controller
  *
- * This call is synchronous. After the BLE Controller is disabled, BLE
+ * This call is synchronous. After the SoftDevice Controller is disabled, BLE
```

Comment:
BLE => Bluetooth? Also there is other parts of the controller that would be unavailable, right? Flash/clock?

Reply:
Yes, BLE should always be spelled out (Bluetooth Low Energy or Bluetooth LE).

Reply:
SoC APIs should be available after disable. Clock APIs are exposed by MPSL